docs/001_develop/03_client-capabilities/008_interaction/stepper.mdx

```diff
@@ -0,0 +1,220 @@
+---
+title: 'Stepper'
+sidebar_label: 'Stepper'
+id: client-interaction-stepper
+keywords: [interaction, interactive, input, stepper, form, data]
+tags:
+- interaction
+- interactive
+- input
+- stepper
+- form
+- data
+sidebar_position: 8
+---
+
+import StepperDemo from '../../../../examples/ui/client-capabilities/interaction/stepper.js';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Stepper
+
+Use cases:
+* Provide a multi-step process or dynamic form that the user can follow.
+
+Rapid Stepper requires `rapid-stepper-tab` and `rapid-stepper-tab-panel` in order to work properly.
```

Comment:
Can you add a description here explaining that the order of the tab corresponds to the order of the tab panel?